buildSrc/src/main/java/com/uber/okbuck/core/dependency/BaseExternalDependency.java

```diff
@@ -84,7 +87,11 @@ public String packaging() {
   public String targetName() {
     StringBuilder targetName = new StringBuilder(versionless().name());
     if (isVersioned()) {
-      targetName.append(NAME_DELIMITER).append(version());
+      String paddedOVersion = Arrays.stream(version().split("\\."))
```

Comment:
add a new method `paddedVersion()` which is used when padding is enabled otherwise uses the normal `version()`